actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -27,7 +27,9 @@ def options
         @options ||=
           case type
           when :chrome
-            ::Selenium::WebDriver::Chrome::Options.new
+            ::Selenium::WebDriver::Chrome::Options.new(
+              args: %w[ --disable-search-engine-choice-screen ]
```

Comment:
These arguments should be setup the same way all the other default arguments are set up. You'll need a new method for chrome and a new case statement. This isn't correct because it doesn't follow the design I implemented for this functionality.